pkgs/by-name/ka/kazv/package.nix

```diff
@@ -0,0 +1,69 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  boost,
+  cmake,
+  cmark,
+  cryptopp,
+  extra-cmake-modules,
+  immer,
+  kdePackages,
+  lager,
+  libkazv,
+  nlohmann_json,
+  olm,
+  pkg-config,
+  qt6,
+  zug,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "kazv";
+  version = "0.5.0";
+
+  src = fetchFromGitLab {
+    domain = "lily-is.land";
+    owner = "kazv";
+    repo = "kazv";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-WBS7TJJw0t57V4+NxsG8V8q4UKQXB8kRpWocvNy1Eto=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    kdePackages.extra-cmake-modules
+    pkg-config
+    qt6.wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    boost
+    cmark
+    cryptopp
+    immer
+    kdePackages.kio
+    kdePackages.kirigami
+    kdePackages.kirigami-addons
+    kdePackages.knotifications
+    lager
+    libkazv
+    nlohmann_json
+    olm
+    qt6.qtbase
+    qt6.qtmultimedia
+    qt6.qtwayland
```

Comment:
qtimageformats is a recommended runtime dependency, without which it cannot display certain images like webp.

Reply:
Added